language/tools/move-cli/src/main.rs

```diff
@@ -32,16 +32,19 @@ use structopt::StructOpt;
 #[structopt(name = "Move", about = "CLI frontend for Move compiler and VM")]
 struct Move {
     /// Directory storing Move resources, events, and module bytecodes produced by script execution.
-    #[structopt(name = "move-data", long = "move-data", default_value = MOVE_DATA)]
+    #[structopt(name = "move-data", long = "move-data", default_value = MOVE_DATA, global = true)]
```

Comment:
This nifty `global` option allows you to use the flag either before or after the subcommand (e.g., `move -v check` or `move check -v`). Without the flag, only the first one works. I added this for all the generic flags because it's pretty annoying without it.